### DISCUSSION TITLE
Zephirus M16 GU603ZX - is it possible to disable iGPU?

### DISCUSSION THREAD

**Discussion by noteMASTER11** [seerge/g-helper#2615] [General]
Hello, I have device GU603ZX (brand new condition) with the following specs:

1. CPU - Intel Core i9-12900H
2. GPU - Intel Iris Xe + RTX 3080Ti 16 GB
3. 2K 165Hz display with lots of certified stuff I don't understand.
4. RAM - DDR5 4800 MHz, 2*16 GB
5. SSD - 980 Pro 1 TB
6. The latest official 311 BIOS of course.

And I kindly ask you to answer the following questions:

1. [**_Solved via reinstalling to the newest ASUSSystemControlInterface Driver_**]

> Why isn't it possible to **edit Fan tables**? G-Helper told me that my motherboard doesn't support such things.

2. Is it possible to **fully disable iGPU Iris Xe** due to the MUX Switch which is in this laptop model?

Thank you for the greatest tool ever, i love it. <3

**Problem detailed description**
It could help with the understanding why I'm trying to apply tweaks to the iGPU and Fan tables.

1. GU603ZX is pretty hot while I'm playing the games such as HDII. Even if I set TDP in the ThrottleStop with the lower options (such as 90 Wt instead of stock 115 Wt Boost) the CPU can reach 92-94 C after the short FPU 10-min stress in AIDA64.
2. The GPU despite its 120 Wt TDP is very hot also. After short MSI Kombustor Donut 6500 MB test in FHD the GPU-Z shows me Thrm bottleneck instead of regular Pwr when Hotspot reaches 96 C. Of course I could edit Curve tables in the Afterburner but both FPS and frequencies (GPU@vRAM) are extremely low -  and that made me upset. :(

![image_2024-05-18_23-39-53 (2)](https://github.com/seerge/g-helper/assets/20218573/0d2c2e87-7851-47b3-94b2-552bf17a0e56)

So I'm trying to reach both dGPU + CPU values that don't overheat my lovely laptop.

**Comment by noteMASTER11** [comment]
Maybe I could help the project with testing Zephirus M16. Could it be appropriate to ask for the support [via the request](https://github.com/seerge/g-helper/issues/new?assignees=&labels=&projects=&template=peripherals_request.yml)?

![2024-05-20 23_50_02-1 204 966 489 - AnyDesk](https://github.com/seerge/g-helper/assets/20218573/63a6efbb-78f4-43b2-a2eb-47a608d75367)
For example, custom Fan tables are working properly without any additions and despite M16 is excluded from the Support List.

**Comment by seerge** [comment]
> Is it possible to fully disable iGPU Iris Xe due to the MUX Switch which is in this laptop model?

iGPU can't be "disabled", as it's part of your APU (CPU + iGPU). 

But when you set ``Ultimate`` mode your internal display will be routed to dGPU directly completely bypassing iGPU.

**Comment by noteMASTER11** [comment]
@seerge Thank you, I see that iGPU activity is always 0% in the Ultimate Mode. But unfortunately it doesn't mean that it's not powered. It is consuming a small part of CPU TDP I suppose.

**Reply by IceStormNG** [reply]
We're talking about 0.1W or even less when it is not in use. If you think disabling it will give more performance to the CPU, then no, it will not.

This is the power draw of the Iris XE when it is in use with desktop usage. When it is idle and has not to drive a display, it will probably go below 0.1W.
![image](https://github.com/seerge/g-helper/assets/12786283/0743a91e-3472-431a-b383-3fb4ee365777)
